File: Code/MestraTest/UnitTestMestraApp/UnitTest1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
using MestraApp;

namespace UnitTestMestraApp
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            Class1 x;
            x.F();

        }
    }
}

File: Code/MestraTest/UnitTest1/unittest1.cpp
#include "stdafx.h"
#include "CppUnitTest.h"
#include "../../../../Stm32/Stm32CubeProjects/MidiSerialTestDMA2Complete/Src/Mestra/Trigger.h"

using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace UnitTest1
{		
	TEST_CLASS(UnitTest1)
	{
	public:
		
		TEST_METHOD(TestMethod1)
		{
			Mestra::Trigger trigger;
			Assert::AreEqual(52, trigger.F());
		}

	};
}